UPGRADE-API-1.12.md

```diff
@@ -99,4 +99,4 @@ Here is how the response looks like:
 
 1. The 2nd parameter `localeCode` has been removed from `src/Sylius/Bundle/ApiBundle/Command/Cart/PickupCart.php` and now is set automatically by `src/Sylius/Bundle/ApiBundle/DataTransformer/LocaleCodeAwareInputCommandDataTransformer.php`.
 
-1. The `api/v2/admin/products` and `api/v2/admin/products/{code}` have been changed and the `defaultVariant` field has been removed.
+1. The responses of endpoints `/api/v2/admin/products` and `/api/v2/admin/products/{code}` have been changed in such a way that the field `defaultVariant` has been removed.
```

Comment:
This line https://github.com/Sylius/Sylius/blob/efc9d781de28c18f9cd1ef778178e863257a1d58/src/Sylius/Bundle/ApiBundle/Swagger/ProductDocumentationNormalizer.php#L41 should also be deleted.